src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+const PRICE_FOR_DAY = 40;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let i = 0; i < days; i++) {
+    res += PRICE_FOR_DAY;
+  }
+
+  if (days >= LONG_TERM) {
+    res -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    res -= SHORT_TERM_DISCOUNT;
```

Comment:
The `else` clause is not needed here because the `if` statement already contains a `return`. Simplify the control flow by removing the `else` keyword and unindenting its block.